automatic/curl/README.md

```diff
@@ -0,0 +1,7 @@
+# <img src=""></img> [curl](https://chocolatey.com/packages/curl)
```

Comment:
or are you referring to the newly added optimize function?
Then no, only jpg and png files.